.github/workflows/openapi-spec.yml

```
@@ -0,0 +1,48 @@
+name: Generate OpenAPI Spec
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)